packages/react-component-playground/webpack.config.js

```diff
@@ -35,22 +35,22 @@ module.exports = {
     'react-dom': 'react-dom',
   },
   resolve: {
-    extensions: ['', '.js', '.jsx'],
+    extensions: ['.js', '.jsx'],
   },
   module: {
```

Comment:
It also continues to support `loaders`, which is great because it allows Cosmos to build webpack configs that work universally without version checking.

That being said, I can start using `rules` here because it's part of the build process, not lib code (same with examples).